docs/Makefile

```diff
@@ -50,7 +50,7 @@ deploy: zip ## Deploy docs to Amplify
 	bash ./deploy-docs.sh public.zip
 
 serve: submodule build ## Boot the development server.
-	hugo server --buildFuture --baseUrl http://127.0.0.1
+	hugo server --buildFuture --baseURL http://127.0.0.1
```

Comment:
This change is in chrisdoherty4's PR: https://github.com/aws/eks-anywhere/pull/7023/files. Probably doesn't matter. And you did find it twice.

Reply:
Yeah this had been bothering me for a while, I thought I'd include the change in my next docs PR and here it is.